.github/workflows/new-release.yml

```diff
@@ -0,0 +1,43 @@
+name: "New plugin release"
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+
+jobs:
+  make-new-release:
+    if: github.event.base_ref == 'refs/heads/master'
+    name: "Releasing plugin"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Restore Gradle caches
+        uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-cache-${{ hashFiles('**/*.gradle.kts') }}-${{ hashFiles('**/gradle.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-cache-
+      - name: Restore Gradle wrapper
+        uses: actions/cache@v1
+        with:
+          path: ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-wrapper-${{ hashFiles('**/gradlew') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-wrapper-
+      - name: Publish plugin
+        env:
+          GRADLE_PUBLISH_KEY: {{ secrets.GradlePublishKey }}
+          GRADLE_PUBLISH_SECRET: {{ secrets.GradlePublishSecret }}
+          GITHUB_KEY: {{ secrets.GithubKey }}
```

Comment:
@JLLeitschuh please check here secrets names

Reply:
Should be `GRADLE_PUBLISH_KEY` and `GRADLE_PUBLISH_SECRET`. I never setup a GitHub key. Isn't there one that's implicit in the environment?

Reply:
I don't see anything related to `GITHUB_KEY` here: https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables

Reply:
I've used proper secret name for `GRADLE_PUBLISH_KEY` and `GRADLE_PUBLISH_SECRET` via force-push.